certified-connectors/GMO Sign/Readme.md

```diff
@@ -0,0 +1,18 @@
+# Title

```

Comment:
@GMO-GSHD-APIGW please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1).